src/witan/models/load_data.clj

```diff
@@ -80,6 +80,21 @@
   {:column-names (apply-col-names-schema PopulationSchema csv-data)
    :columns (vec (apply-row-schema PopulationSchema csv-data))})
 
+(defmethod apply-rec-coercion :births
```

Comment:
We're coercing the data before it is a dataset (when it is still in vectors), using a coercion fn and the row schemas that you helped us make from col schemas. I'm inclined to leave this for now & talk to you about improvements in the future.